### PR TITLE
[RW-4981][risk=no] make all crons return 204 on success

### PR DESCRIPTION
**Pull request by jaycarlton**
It doesn't appear that we can use the payload from a cron endpoint, and it's better to have everything return 204 for monitoring purposes (so you don't confuse the two that are 200s).

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI + API server with `yarn test-local`
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/api/OfflineClusterController.java [hunk 1]
```diff
@@ -65,16 +64,16 @@
    * <p>As an App Engine cron endpoint, the runtime of this method may not exceed 10 minutes.
    */
   @Override
-  public ResponseEntity<CheckClustersResponse> checkClusters() {
-    Instant now = clock.instant();
-    WorkbenchConfig config = configProvider.get();
-    Duration maxAge = Duration.ofDays(config.firecloud.clusterMaxAgeDays);
-    Duration idleMaxAge = Duration.ofDays(config.firecloud.clusterIdleMaxAgeDays);
+  public ResponseEntity<Void> checkClusters() {
+    final Instant now = clock.instant();
```
Comment (jaycarlton) [inline api/src/main/java/org/pmiops/workbench/api/OfflineClusterController.java:68]: The finalpalooza here is to highlight the not-final counting vars.

File: api/src/main/java/org/pmiops/workbench/billing/OfflineBillingController.java [hunk 2]
```diff
@@ -38,16 +35,6 @@
   @Override
   public ResponseEntity<Void> bufferBillingProjects() {
     billingProjectBufferService.bufferBillingProjects();
-    // TODO(jaycarlton) we're going to need a richer framework for cron reporting,
```
Comment (jaycarlton) [inline api/src/main/java/org/pmiops/workbench/billing/OfflineBillingController.java:41]: This metric was no longer in use now that we have a general operation time metric.

File: api/src/test/java/org/pmiops/workbench/api/OfflineClusterControllerTest.java [hunk 3]
```diff
@@ -61,16 +54,20 @@ Clock clock() {
...
       config.firecloud.clusterMaxAgeDays = (int) MAX_AGE.toDays();
       config.firecloud.clusterIdleMaxAgeDays = (int) IDLE_MAX_AGE.toDays();
       return config;
     }
   }
 
-  @Autowired ClusterApi clusterApi;
-  @Autowired OfflineClusterController controller;
+  @Qualifier(NotebooksConfig.SERVICE_CLUSTER_API)
+  @MockBean private ClusterApi mockClusterApi;
```
Comment (jaycarlton) [inline api/src/test/java/org/pmiops/workbench/api/OfflineClusterControllerTest.java:66]: I don't actually understand how this worked before. We were effectively treating it like a mock and it was happy, but it wasn't declared as a mock...

Comment (calbach) [reply api/src/test/java/org/pmiops/workbench/api/OfflineClusterControllerTest.java:66]: It was declared as a mock...

Comment (jaycarlton) [reply api/src/test/java/org/pmiops/workbench/api/OfflineClusterControllerTest.java:66]: Oh I see. That makes sense.

**Pull request merged**